### ISSUE TITLE
Fresh CentOS qemu-img failed with qemu-img: Unable to initialize gcrypt

### ISSUE THREAD

**Issue by rbo** [RedHat-EMEA-SSA-Team/hetzner-ocp4#160]
```
TASK [openshift-4-cluster : Create disk for demo-bootstrap] *******************************************************************************************************************************
Monday 21 December 2020  08:43:07 +0100 (0:00:05.419)       0:00:39.168 *******
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["qemu-img", "create", "-f", "qcow2", "-b", "/var/lib/libvirt/images/rhcos-4.6.1-x86_64-qemu.x86_64.qcow2", "/var/lib/libvirt/image
s/demo-bootstrap.qcow2", "120G"], "delta": "0:00:00.004845", "end": "2020-12-21 08:43:07.181992", "msg": "non-zero return code", "rc": 1, "start": "2020-12-21 08:43:07.177147", "stderr":
"qemu-img: Unable to initialize gcrypt", "stderr_lines": ["qemu-img: Unable to initialize gcrypt"], "stdout": "", "stdout_lines": []}

```

**Comment by rbo** [comment]
Looks like we have to update libcrypt: https://bugzilla.redhat.com/show_bug.cgi?id=1828681

**Comment by rbo** [comment]
`dnf update` solve the problem.

**Comment by rbo** [comment]
On RHEL 8 we run 
```
- name: Upgrade all packages
  yum:
    name: '*'
    state: latest
```
But not on CentOS 8 :-(

**Comment by rbo** [comment]
Fixed in devel branch, will be merged into master via PR #159
